charts/consul/Chart.yaml

```diff
@@ -13,14 +13,14 @@ sources:
   - https://github.com/hashicorp/consul
   - https://github.com/hashicorp/consul-k8s
 annotations:
-  artifacthub.io/prerelease: true
+  artifacthub.io/prerelease: false
   artifacthub.io/images: |
     - name: consul
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.19-dev
```

Comment:
Why are we moving from `docker.mirror.hashicorp.services/hashicorppreview/` to `hashicorp/consul`?

Reply:
`make prep-release`does this.  `make prep-release-dev` sets it to hashicorp preview.